r/R/rzeppelin/R/protocol.R

```diff
@@ -0,0 +1,35 @@
+UNSUPPORTED_TYPE <- 0L
```

Comment:
It's [rscala](https://github.com/cran/rscala/blob/master/R/protocol.R) [BSD](https://cran.r-project.org/web/packages/rscala/index.html)